frontend/src/services/profile_handler_service.ts

```diff
@@ -14,24 +14,23 @@ export class FailureToUpdateProfile extends Error {
   }
 }
 
-interface ProfileHandlerServiceInterface {
-  getPerson(id: string): Promise<string>;
-  updatePerson(PersonProps): Promise<boolean>;
-  deletePerson(id: string): Promise<boolean>;
+interface BackendServiceInterface {
```

Comment:
nit: This should live in a different file. In general it might be nice to have a folder for each service, that way `backend_service_interface` and `mock_backend_service` and `backend_service_test` and any other backendy files can live together